packages/airnode-examples/integrations/config-utils.ts

```diff
@@ -65,9 +66,30 @@ export const createNodeVersion = () => {
   return packageVersion;
 };
 
-export const generateConfigFile = (dirname: string, config: Config, generateExampleFile: boolean) => {
+/**
+ * If the AirnodeRrpV0DryRun contract has been deployed, add its address to the config file,
+ * otherwise, return the config file as is.
+ *
+ * Note when using a public blockchain an AirnodeRrpV0DryRun deployment is not necessary as we default
+ * to using the API3 deployment
+ */
+export const addAirnodeRrpV0DryRunAddress = async (config: Config) => {
+  const goAirnodeRrpDryRun = await go(() =>
+    getDeployedContract('@api3/airnode-protocol/contracts/rrp/AirnodeRrpV0DryRun.sol')
+  );
+  if (!goAirnodeRrpDryRun.success) {
+    return config;
+  }
+  const airnodeRrpDryRun = goAirnodeRrpDryRun.data;
+  /* eslint-disable-next-line functional/immutable-data */
+  config.chains[0].contracts = { ...config.chains[0].contracts, AirnodeRrpDryRun: airnodeRrpDryRun.address };
+  return config;
```

Comment:
```suggestion
return {
    ...config,
    chains: [
      { ...config.chains[0], contracts: { ...config.chains[0].contracts, AirnodeRrpDryRun: airnodeRrpDryRun.address } },
    ],
  };
```

Maybe we can use this bad looking suggestion for the sake of immutability 😄

Reply:
Haha ok 👍